src/en/space-station-14/departments/service/proposals/theshued-librarian-gameplay.md

```diff
@@ -0,0 +1,160 @@
+
+The librarian is a role about learning about books and interacting with people. 
+On its own, the librarian is only concerned with processing information and distributing it.
+To do his job, he **must** communicate and negotiate with other people. 
+
+### Multidisciplinary
+The librarian's activities should be able to interact with all departments in the game in two ways:
+1) The departments have the opportunity to assist the librarian in his work
+2) The librarian's work can improve the quality of life of the departments.
+
+This also applies to antagonists. The librarian can be helped by the antagonists, and the librarian's work can help the antagonists.
+
+### Non-deterministic
+
+Given the concept of working with information, for the following principles to work, this "information" must be randomized each round.
+
+
+
+## Implementation
+
+### Encrypted books
+
+The librarian's entire gameplay will revolve around encrypted books.
+
+All books initially have encrypted titles and a random appearance, which does not allow you to immediately understand what kind of knowledge it is.
+But under the encrypted appearance is one of the specific books that have the following parameters:
+| Parameter | Type | Description |
+|---|---|---|
+| Discipline | List<Enum> | All books relate to specific well-defined disciplines. Books can be related disciplines. This affects which department can help you study this book. |
+| Legality | Bool | Illegal books carry dangerous knowledge, and the security service needs to control what exactly is being studied in order to prevent the leakage of dangerous knowledge to ordinary personnel. |
+| Result recipe | ProtoId | Books consists of a recipe for items that have randomized crafting each round. The successful decoding of the book is a literary story with a description of the crafting sequence of the item. |
+
+This is an alternative way to explore powerful equipment that does not concern the scientists department.
+The items unlocked by books are thematically more related to "mystical" "alien" and "homemade" than to "high-tech"
+The book can contain recipes for crafting 1 or many item, or complex structures.
+
+Also, this category should be a good source of references to existing fantasy or sci-fi books.
+To begin with, the following items are proposal:
+
+| Book | Description | Discipline keywords | Legality |
+|---|---|---|---|
+| The legend of the Dagger of Withering | A special melee weapon that sprays Plant-B-Gone during attacks. It is extremely effective for fighting kudzu. | 3 Biology + 2 Military + 1 Mysticism| ✔️
+| The story of wizards with silly magic wands and owls | A reference to Harry Potter, a self-loading ranged weapon that shoots rainbow projectiles with minor damage. | 7 Mysticism + 3 Military | :x:
+| The Stargate | Unlocks a complex crafting sequence that allows engineers to create gates for teleportation to planets (Current Gateways) | 18 Engineering + 2 Musticism | ✔️
```

Comment:
"Musticism"

Other than that seems like a pretty good idea. I do wonder how other departments would get the knowledge of keywords, like you say engineers would get 10 tech keywords. How would that work mechanically, would they be given at the start of the round like syndicate codewords? (I wouldn't like how overwhelming this might feel esp. to new players and this seems like a high cost for a department that may never even see the librarian, depending on how the round goes)

Reply:
The main idea was not to dump 10 useless code words on players, but that they passively decipher them when they see them in a book or chat. But that's the questionable part overall.